src/functions/content-types/product/item.ts

```diff
@@ -80,3 +82,19 @@ function normalizeToCM(volume, unit) {
       throw new Error(`Unsupported volume unit: ${unit}`);
   }
 }
+
+function calculateNeedsMetFields(data) {
+  const { items, months, people } = data;
+  let monthlyNeedsMetPerItem = (people * months) / items;
+  monthlyNeedsMetPerItem = parseFloat(monthlyNeedsMetPerItem.toFixed(2));
```

Comment:
@madcalf 

* **question(blocking):** is there a specific reason we're limiting this to 2 digits of precision?
* **question(blocking):** do you need the `parseFloat`?  `monthlyNeedsMetPerItem` should be a float already, and `.toFixed` [returns a float](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toFixed).

Reply:
Ah no actually meant to ask about this. I really was just wanting it to _display_ 2 digits of precision like in the spreadsheet but wasn't sure how to have the Admin Panel to display a different precision than the actual value.
And the `parseFloat` just keeps `monthlyNeedsMetPerItem` a number because applying `toFixed()` converts it to a string. 

I'll remove this precision limit.

Reply:
@madcalf ah yeah, you're totally right about `toFixed()`, I misread the docs there!  But agreed on getting rid of the precision limit- we'll have to find another way to handle the display in the admin panel but it should be fine for now.